src/Bicep.Decompiler.IntegrationTests/Files/Working/101-1vm-2nics-2subnets-1vnet/azuredeploy.bicep

```diff
@@ -80,6 +80,7 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2019-12-01' = {
       bootDiagnostics: {
         enabled: true
         storageUri: reference(diagStorageAccount.id, '2019-06-01').primaryEndpoints.blob
+//@[20:66) [use-resource-symbol-reference (Warning)] Use a resource reference instead of invoking function "Bicep.Core.Syntax.IdentifierSyntax". This simplifies the syntax and allows Bicep to better understand your deployment dependency graph. (CodeDescription: bicep core(https://aka.ms/bicep/linter/use-resource-symbol-reference)) |reference(diagStorageAccount.id, '2019-06-01')|
```

Comment:
"Bicep.Core.Syntax.IdentifierSyntax"?  That seems rather confusing, can we simplify?

Reply:
Good catch, that wasn't meant to happen! Will fix this.